dev/set_matrix.py

```diff
@@ -338,6 +339,11 @@ def expand_config(config):
         flavor = get_flavor(name)
         package_info = PackageInfo(**cfgs.pop("package_info"))
         all_versions = get_released_versions(package_info.pip_release)
+        free_disk_space = package_info.pip_release in (
+            "transformers",
+            "sentence-transformers",
+            "torch",
```

Comment:
Shall we keep the condition `startsWith(matrix.version, '2.')` for torch ?

Reply:
No, I don't think we need to keep it.